src/Controllers/ResultController.php

```diff
@@ -43,9 +41,11 @@ public function getScorePage(RequestInterface $request)
         $userId = $request->getRequestParameters()['userId'];
         $questionInstanceEntities = $this->resultService->getQuestionsInstance($quizInstanceId);
 
-        $textInstnaces = [];
+        $textInstances = [];
         foreach ($questionInstanceEntities as $key => $value) {
```

Comment:
Why does resultService->getTextInstance() return an array

Reply:
This was because of the resultService was searching for the entity using the findBy method instead of findOneBy.